app/assets/javascripts/application.js

```diff
@@ -29,7 +27,7 @@
 //= require govuk_publishing_components/components/radio
 //= require govuk_publishing_components/components/step-by-step-nav
 //= require govuk_publishing_components/components/error-summary
-//= require govuk_publishing_components/analytics/explicit-cross-domain-links
+//= require govuk_publishing_components/analytics
```

Comment:
Removing `govuk_publishing_components/analytics/explicit-cross-domain-links` sounds good, as we already have it in `govuk_publishing_components/analytics.js`, but we shouldn't need this line as `govuk_publishing_components/analytics` is [loaded by `static`](https://github.com/alphagov/static/blob/main/app/assets/javascripts/analytics.js.erb#L1) in all templates.